### ISSUE TITLE
Curator failing silently with invalid yaml

### ISSUE THREAD

**Issue by fbacchella** [elastic/curator#1504]
I'm using curator 5.8.1.

I tried using the following yaml file:
```
---
actions:

  1:
    action: delete_indices
    description: >-
      Delete indices older than 180 days.
    options:
      disable_action: False
      ignore_empty_list: True
      timeout_override:
      continue_if_exception: False,
    filters:
      - filtertype: age
        source: creation_date
        direction: older
        unit: days
        unit_count: 180
        exclude: False
```

It's wrong, look at the comma after the 'continue_if_exception'.
But curator fails silently on it:

```
2020-01-24 11:08:02,962 DEBUG                curator.cli                    run:110  Client and logging options validated.
2020-01-24 11:08:02,962 DEBUG                curator.cli                    run:114  default_timeout = 30
2020-01-24 11:08:02,962 DEBUG                curator.cli                    run:118  action_file: /tmp/infsec.yaml
```

A error message should be returned.

**Comment by untergeek** [comment]
Better YAML linting is worth adding. I will add this when time permits (which isn't often).

**Comment by aggarwalShivani** [comment]
> Better YAML linting is worth adding. I will add this when time permits (which isn't often).
Any plans of adding this in near future :) ?

**Comment by aggarwalShivani** [comment]
@untergeek Hi, any timelines when this could be added in curator?

**Comment by aggarwalShivani** [comment]
Hi @untergeek , is fix for this issue being planned in curator? Any timelines when this can be expected?

**Comment by untergeek** [comment]
No timeline. I'm no longer in engineering, or consulting. ILM and SLM have superseded Curator (they even removed Curator from ECE 3.0, announced today). Curator persists as a labor of love, and I simply do not have a lot of spare time.

That doesn't mean I don't want to work on Curator. It just means I'm waiting for the winds to shift a bit at work, which will give me a bit more time to do work like this on the clock, instead of using personal time on it.

**Comment by aggarwalShivani** [comment]
Hi @untergeek,
Now that new curator 6.x, 7.x releases are coming in now, is there a plan to include this fix in 7.x stream?

**Comment by untergeek** [comment]
**TL;DR** — If you're not using an Elastic released version of Curator, a backport to fix this particular issue will almost certainly not get pushed.

Maybe. Backporting is not a big priority right now. Any backport to 7.x (while it will be delayed due to focus on new features and fixes to the 8.x branch) will have a potential downside to users in that it will only work on 7.17.x (and _maybe_ a few of the releases as far back as 7.11 and prior but with no guarantees). The reason is the massive code base shift to separating the client creation functionality. Once I put that change in, releases to the 7.x branch will ONLY work on official, Elastic released images as it will immediately drop all AWS Elasticsearch functionality.
